index-clip.md

```diff
@@ -51,11 +53,28 @@ You can choose between three different types of peak calling for the data analys
 
 **Table 1**: Data specification of the different peak calling algorithms.
 
-| Tool | Biological Replicates (Yes/No) | Control Data (Yes/No) |
-| ---            | :-:      | :-:     |
-| <a href="https://github.com/tbischler/PEAKachu">PEAKachu</a>            | Yes | Yes   |
-| <a href="https://doi.org/10.1186/s13059-017-1364-2">PureCLIP</a>            | No | Yes   |
-| <a href="https://doi.org/10.1093/bioinformatics/bts569">Piranha</a>           | No | No   |
+<table class="table table-striped">
```

Comment:
If you just wanted to make the old table striped you can just include

```
[table stuff]
{: .table.table-striped}
```

Reply:
Oh well xD didn't know about the easier way. I looked into the code of the galaxy webpages which lead me to my solution ^^